Applications/Spire/Include/Spire/Ui/CheckBox.hpp

```diff
@@ -6,11 +6,6 @@
 #include "Spire/Ui/Ui.hpp"
 
 namespace Spire {
-namespace Styles {
-
-  /** Selects a checked CheckBox. */
-  using Checked = StateSelector<void, struct CheckedTag>;
```

Comment:
This is a change to the public API and has not been approved. Renaming ```CheckedTag``` to ```CheckedSelectorTag``` is a good idea, however.